README.md

```diff
@@ -73,7 +73,7 @@ This feature requires botUser mode.
    `https://mydomain.com/hooks/xxxx` where `xxxx` is the integration token and
    `https://mydomain.com/hooks/` is the `Slack compatible app URL`.
 4. Install this plugin on your Jenkins server.
-5. Configure it in your Jenkins job (and optionally as global configuration) and
+5. Configure it in your Jenkins job (and optionally as a [global configuration using an additional plugin](https://github.com/jenkinsci/global-slack-notifier-plugin)) and
```

Comment:
This isn’t quite right, global configuration will set all the values for authentication, channel, and slack instance. You can then override per job or not.

The post build action is the regular adding it manually per job.

The additional plugin is for people that want it to be applied to every job (note I’ve never used the plugin I just know of its existence)

Reply:
Good call, I see where this can create confusion. I just changed the message again now.